test/features/install_dataindex.feature

```diff
@@ -17,3 +17,29 @@ Feature: Kogito Data Index
       }
     }
     """
+
+#####
+
+  @persistence
+  @external-infinispan
+  Scenario: Install Kogito Data Index with persistence using external Infinispan
+    And Infinispan Secret "external-infinispan-secret" is created with credentials:
+      | developer | mypass                      |
```

Comment:
could we use a Scenario Outline with Examples for this username/password ?

Reply:
umm, scenario outline would have sense in case we would execute scenario several times with different parameters, right?
In this case both credentials are applied together, first is for Infinispan "user" (used in Kogito), second is used by Infinispan operator

Reply:
Or do you mean to use Scenario outline to increase readability?

Reply:
no it was just to avoid mistakes but it may indeed increase complexity of reading it.